tests/v3/connection_state_test.c

```diff
@@ -3366,8 +3366,8 @@ AWS_TEST_CASE_FIXTURE(
     &test_data)
 
 /**
- * Makes a CONNECT, with 1 second keep alive ping interval, send a publish roughly every second, and then ensure NO
- * pings were sent
+ * Makes a CONNECT, with 1 second keep alive ping interval, send a different operation every second, and then ensure NO
```

Comment:
If this comment is accurate then I feel like this test is likely overly sensitive to timing conditions and containerization.